services/util/loopy.py

```diff
@@ -78,9 +71,8 @@ def get_next_page(self):
         if self.page_size > self.result_count:
             page = self.result_count
 
-        query_string = self.get_query_string() + \
-           'filter[limit]={}&filter[skip]={}&filter[order]={}'.format(page,
-            self.current_page*self.page_size, self.order_by)
+        query_string = self.get_query_string(page, self.current_page*self.page_size, self.order_by)\
```

Comment:
@justinlueders looks good. would you check that it works with aggregate-clustering module, which is built with python 3. only issue might be with urllib, which changed in 3 I believe. running  ```2to3 loopy.py``` might be helpful too.